var/spack/repos/builtin/packages/python-dev/package.py

```diff
@@ -25,6 +25,7 @@ class PythonDev(Package):
     depends_on('py-ipyparallel', type=('build', 'run'))
     depends_on('py-ipython', type=('build', 'run'))
     depends_on('py-jinja2', type=('build', 'run'))
+    depends_on('py-jinja2-cli', type=('build', 'run'))
```

Comment:
Do I also need to bump `python-dev` version?

Reply:
Oops, forgot to rename `jinja2-cli` -> `py-jinja2-cli` ^^.

Reply:
> Do I also need to bump `python-dev` version?

Since none of the dependent packages have changed, I think we can skip this here. Once Jenkins is done, you should verify that `py-jinja2-cli` is added to the path in
```
/gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/238/modules/serial-libraries/2019-02/tcl/linux-rhel7-x86_64/python-dev/0.1/python3
```